File: MogoDB-LS-Backend/models/Story.js
// models/Story.js
const mongoose = require('mongoose');

const storySchema = new mongoose.Schema({
  storyId: { type: String, required: true, unique: true },
  title: String,
  description: String,
  dateOfFact: Date,
  photoPath: String,
  locationId: String,
  userId: String
});

module.exports = mongoose.model('Story', storySchema);

File: MogoDBAndroid/app/src/main/java/com/localstories/Story.kt
package com.localstories

data class Story(
    val storyId: String,
    val title: String,
    val description: String,
    val dateOfFact: String, // Use String for now, can convert to Date later
    val photoPath: String,
    val locationId: String,
    val userId: String,
    val author: String
)
